core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2268,6 +2268,9 @@ public static PropertyKey fromString(String input) {
   /** Property name. */
   private final String mName;
 
+  /** Property Key description. */
+  private String mDescription;
```

Comment:
Yes, I understand that `create` changes `mDescription`.

I was wondering why isn't `mDescription` just set in the constructor, like `mName`?